docs/manpage.rst

```diff
@@ -387,6 +387,7 @@ Options controlling ReFrame execution
    ReFrame will try to convert ``VAL`` to the type of the variable.
    If it does not succeed, a warning will be issued and the variable will not be set.
    ``VAL`` can take the special value ``@none`` to denote that the variable must be set to :obj:`None`.
+   Furthermore, ``VAL`` can take one of the special ``@True`` and ``@False`` values according to which the variable is set to :class:`True` and :class:`False` respectively.
```

Comment:
Combine all these:

```suggestion
   ``VAL`` can take one of the following special values: 
    - ``@none``: ...
    - ``@true``: ...
    - ``@false``: ...
```

We will also need to point out that if somebody passes any other value to a boolean variable this will be interpreted as true, because ReFrame tries to convert implicitly the given string to boolean, which is a valid operation in Python.